apps/src/applab/Reducers.js

```diff
@@ -0,0 +1,42 @@
+/** @file Redux reducer functions for App Lab.
+ *  @see http://redux.js.org/docs/basics/Reducers.html */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var ActionType = require('./Actions').ActionType;
+
+function rootReducer(state, action) {
+  if (typeof state === 'undefined') {
+    state = {
+      assetUrl: function () {}
+    };
+  }
+
+  switch (action.type) {
+    case ActionType.SET_LEVEL_PROPS:
+      var newState = $.extend({}, state);
+      [
+        'assetUrl',
+        'isDesignModeHidden',
+        'isEmbedView',
+        'isReadOnlyWorkspace',
+        'isShareView',
+        'isViewDataButtonHidden'
+      ].forEach(function (propName) {
+        if (typeof action.props[propName] !== 'undefined') {
+          newState[propName] = action.props[propName];
+        }
+      });
+      return newState;
```

Comment:
If I'm going to use `_.pick` then I might as well use `_.assign` too, instead of `$.extend`.  Neither pick nor assign are currently part of our custom lodash build.  Do I:

1. Add pick and assign to our lodash build?
1. Just add pick and use with with extend?
1. Finally include all of lodash?